tests/ha/single_pod_restart/test_data_pod_restart.py

```diff
@@ -2931,84 +2901,48 @@ def test_continuous_reads_during_pod_restart_ros(self):
+        self.deployment_backup = resp[1][pod_name]['deployment_backup']
+        self.restore_method = resp[1][pod_name]['method']
+        assert_utils.assert_true(resp[0], "Cluster/Services status is not as expected")
+        LOGGER.info("Step 2: Successfully shutdown data pod %s. Verified cluster and "
+                    "services states are as expected & remaining pods status is online.", pod_name)
         self.restore_pod = True
-        self.restore_method = const.RESTORE_DEPLOYMENT_K8S
-
-        LOGGER.info("Step 3: Check cluster status")
-        resp = self.ha_obj.check_cluster_status(self.node_master_list[0])
-        LOGGER.debug("Response: %s", resp)
-        assert_utils.assert_false(resp[0], resp)
-        LOGGER.info("Step 3: Cluster is in degraded state")
-
-        LOGGER.info("Step 4: Verify services that were running on pod %s are in offline state",
-                    pod_name)
-        resp = self.hlth_master_list[0].get_pod_svc_status(pod_list=[pod_name], fail=True,
-                                                           hostname=pod_host)
-        LOGGER.debug("Response: %s", resp)
-        assert_utils.assert_true(resp[0], resp)
-        LOGGER.info("Step 4: Verified services of %s are in offline state", pod_name)
-
-        remain_pod_list = list(filter(lambda x: x != pod_name, pod_list))
-        LOGGER.info("Step 5: Verify services status on remaining pods %s are in online state",
-                    remain_pod_list)
-
-        resp = self.hlth_master_list[0].get_pod_svc_status(pod_list=remain_pod_list,
-                                                           fail=False)
-        LOGGER.debug("Response: %s", resp)
-        assert_utils.assert_true(resp[0], resp)
-        LOGGER.info("Step 5: Services of pod are in online state")
-
-        LOGGER.info(
-            "Step 6: Perform READs and verify DI on the written data in background during "
-            "pod restart using %s method", self.restore_method)
+        LOGGER.info("Step 3: Perform READs and verify DI on the written data in background during "
+                    "pod restart using %s method", self.restore_method)
         args = {'s3userinfo': list(users.values())[0], 'log_prefix': self.test_prefix,
                 'nclients': 2, 'nsamples': 20, 'skipwrite': True, 'skipcleanup': True,
                 'output': output, 'setup_s3bench': False}
-
         thread = threading.Thread(target=self.ha_obj.event_s3_operation,
                                   args=(event,), kwargs=args)
         thread.daemon = True  # Daemonize thread
         thread.start()
         # TODO Need to update timing once we get stability in degraded IOs performance
         time.sleep(HA_CFG["common_params"]["degraded_wait_delay"])
-        LOGGER.info("Step 6: Successfully started READs and verified DI on the written data in "
+        LOGGER.info("Step 3: Successfully started READs and verified DI on the written data in "
                     "background")
-
```

Comment:
So because this is ROS test, we are not doing any writes in degraded mode right?